### PR TITLE
Fufu sensory traits

### PR DESCRIPTION
**Pull request by marieALaporte**
Description 
-----------------------------------------------------

Checklist 
---------------------------------------------------------------------------------------
- [ ] An OBO file generated from an OWL file
- [X] New terms
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] Term updates
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] CHECKS
  - [ ] New variables have a parent trait ID 
    - [ ] New variables have methods and scales 
  - [X] The OBO file has been validated
    - [ ] checked in cassavabase
    - [X] checked CropOntology

### CODE REVIEW

**Review by nmenda — APPROVED** [review]

File: cassava_trait.obo [hunk 1]
```diff
@@ -7594,6 +7594,139 @@ def: "Aroma like tha of sweet potato, Possibly like aroma of boiled, roasted or
...
+[Term]
+id: CO_334:0003052
+name: fufu stretchability in hand
+namespace: CassavaTrait
+def: "Ability of fufu to stretch when pulled apart with fingers" []
+synonym: "FufHdStret" EXACT []
+is_a: CO_334:0103019 ! Texture in hand trait
+
+[Term]
+id: CO_334:0003053
```
Comment (nmenda) [inline cassava_trait.obo:7723]: should this be consolidated with 'fufu smell'? 
https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L6496

File: cassava_trait.obo [hunk 2]
```diff
@@ -7594,6 +7594,139 @@ def: "Aroma like tha of sweet potato, Possibly like aroma of boiled, roasted or
...
+namespace: CassavaTrait
+def: "Meassures the easiness of fufu to stick together without disintegrating easiness to mould." []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003050 ! fufu moldability in hand
+relationship: variable_of CO_334:0013008 ! texture by handfeel method
+relationship: variable_of CO_334:0103025 ! Moldability scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003044
```
Comment (nmenda) [inline cassava_trait.obo:7642]: is this similar to this scale? 
https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L12027

File: cassava_trait.obo [hunk 3]
```diff
@@ -7594,6 +7594,139 @@ def: "Aroma like tha of sweet potato, Possibly like aroma of boiled, roasted or
...
+def: "Measures the adherence of fufu onto the fingers when it is touched with fingers or during moulding into bolus for swallowing" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003049 ! fufu stickiness in hand
+relationship: variable_of CO_334:0013007 ! texture by handfeel method
+relationship: variable_of CO_334:0103024 ! Stickiness scale 0-10
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003043
+name: Fufu moldability scale 0-10
```
Comment (nmenda) [inline cassava_trait.obo:7632]: existing scale? 
https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L12076

File: cassava_trait.obo [hunk 4]
```diff
@@ -7594,6 +7594,139 @@ def: "Aroma like tha of sweet potato, Possibly like aroma of boiled, roasted or
...
+def: "Measures the intensity of the different colours of the cooked fufu surface on a scale 1 to 8" []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003047 ! fufu surface colour
+relationship: variable_of CO_334:0013005 ! appearance method
+relationship: variable_of CO_334:0103022 ! Colour scale 1-8
+created_by: Ugo Chijioke
+
+[Term]
+id: CO_334:0003041
+name: Fufu smoothness scale 0-10
```
Comment (nmenda) [inline cassava_trait.obo:7610]: there are 3 smoothness 1-10 scales. Need to make each one have a unique name or consolidate if the same 

https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L12155

https://github.com/Planteome/CO_334-cassava-traits/blob/3b05da7d5a44d0b02f8f74702c66329f5f4c78c4/cassava_trait.obo#L12020

**Review by nmenda — CHANGES_REQUESTED** [review]
check redundant scale and trait terms.
Which ones need to be changed so the names are unique and which ones need to be merged.

**Comment by marieALaporte** [comment]
thanks @nmenda. Let me check with Amos and come back to you

File: cassava_trait.obo (hunk 2 above)
Comment (marieALaporte) [reply cassava_trait.obo:7642]: we will merge the IDs as we agree that they are the same: a 0-10 scale for hardness, even if the products are different

File: cassava_trait.obo (hunk 3 above)
Comment (marieALaporte) [reply cassava_trait.obo:7632]: These scale are different as fufu being a dough, it cannot crumble as boiled cassava do for instance. I will make the names reflect that.

File: cassava_trait.obo (hunk 1 above)
Comment (nmenda) [reply cassava_trait.obo:7723]: I'll wait for the commit and then will merge this and update Cassavabase. Thanks!

**Comment by nmenda** [comment]
@marieALaporte 
I fixed the namespace for the new variables to cassava_trait and made the variable names unique (some were similar to the scale names and it can be an issue for those who load both traits and scales in Chado)

**Review by nmenda — APPROVED** [review]

**Pull request merged**